packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -91,24 +94,26 @@ export class GuxSelectorCard {
 
   render(): JSX.Element {
     return (
-      <Host
+      <div
         class={{
+          'gux-selector-card': true,
           [`gux-${this.variant}`]: true,
           'gux-disabled': this.disabled
         }}
+        onClick={() => this.input.setAttribute('checked', 'true')}
       >
         <div class="gux-content">
           <div class="gux-icon">
             <slot name="icon" />
           </div>
-          <gux-truncate max-lines={2}>
+          <gux-truncate class="gux-label" max-lines={2}>
             <slot name="label" />
           </gux-truncate>
           <slot name="input" />
-          {this.renderContent()}
+          {this.renderDescription()}
           {this.renderBadge()}
         </div>
-      </Host>
+      </div>
```

Comment:
@daragh-king-genesys @gavin-everett-genesys I have a problem with this. As it currently stands, I have the input running over everything with a `position: absolute;`. The idea of this was to avoid adding a `checked` state property which has to be updated in JavaScript based on the input. This is so the card border can change color appropriately. 

However, doing it this way blocks the tooltip for truncated text from showing because the stacking context gives priority to the `input`. 

Do either of you know of any way I can make the tooltips available without going back to a `checked` state type thing (to support changing the card border when input is selected)? I've looked at a bunch of stuff but nothing is jumping out at me so far. Any thoughts or rough guess appreciated.

Reply:
Will check this out 👍

Reply:
If you just apply a higher `z-index` to the truncate component would this work without anything breaking for example a `z-index:2` potentially ?

Reply:
I spent so long thinking about that and not getting the z-index to work. I think it should work now. Thanks Gavin!